packages/runtime/src/components/builtin/constants.ts

```diff
@@ -13,6 +13,7 @@ export const MakeswiftComponentType = {
   Image: './components/Image/index.js',
   Navigation: './components/Navigation/index.js',
   Root: './components/Root/index.js',
+  Slot: './components/Slot/index.js',
```

Comment:
The path in the types is an artifact of the original paths of the builtin components before we event had custom hosts. Let's instead use something that's not a path. How about `makeswift::components::slot::v1`?